eth/tracers/native/prestate.go

```diff
@@ -98,6 +99,13 @@ func (t *prestateTracer) CaptureStart(env *vm.EVM, from common.Address, to commo
 	t.lookupAccount(from)
 	t.lookupAccount(to)
 	t.lookupAccount(env.Context.Coinbase)
+	// [Kroma: START]
+	if env.ChainConfig().Kroma != nil {
```

Comment:
If this is for the Migrate tracer, it would be nice to have a condition ( < KromaMPTTime )